app/jobs/risc_delivery_job.rb

```diff
@@ -100,4 +100,8 @@ def faraday
   def inline?
     queue_adapter.is_a?(ActiveJob::QueueAdapters::InlineAdapter)
   end
+
+  def self.warn_only? 
+    true
```

Comment:
In this case, I think we only want to be warning when it's one of the retry errors. We'd still want to be alerted if it failed for another reason 🙂

Reply:
Ah ok. Making updates.. :)

Reply:
Check now?